suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+type DiscoveryConfigState = {
+    areTestnetsEnabled: boolean;
+};
+
+type DiscoveryConfigSliceRootState = {
+    discoveryConfig: DiscoveryConfigState;
+};
+
+const discoveryConfigInitialState: DiscoveryConfigState = {
+    areTestnetsEnabled: false,
+};
+
+export const discoveryConfigPersistWhitelist: Array<keyof DiscoveryConfigState> = [
+    'areTestnetsEnabled',
+];
```

Comment:
This is not used anywhere apart from reducers.ts - I would just follow the convention we now have with persisted keys and use it there if you're not against that